### PR TITLE
nginx-configmap - adding portal route to nginx resolver

### PR DESCRIPTION
**Pull request by Alevsk**

### CODE REVIEW

File: cluster/nginx.go [hunk 1]
```diff
@@ -250,10 +250,25 @@ func UpdateNginxConfiguration(ctx *Context) chan error {
 																		'$status $body_bytes_sent "$http_referer" '
 																		'"$http_user_agent" "$http_x_forwarded_for"';
 				server {
-					#listen 80 default_server;
-					#listen 443 ssl default_server;
-					server_name _ ;
-					return 404;
+					location / {
```
Comment (dvaldivia) [inline cluster/nginx.go:253]: what about /api/ ?

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: cluster/nginx.go [hunk 2]
```diff
@@ -250,10 +250,25 @@ func UpdateNginxConfiguration(ctx *Context) chan error {
...
+						 proxy_set_header X-Frame-Options SAMEORIGIN;
+						 proxy_buffers 256 16k;
+						 proxy_buffer_size 16k;
+						 client_body_timeout 60;
+						 send_timeout 300;
+						 lingering_timeout 5;
+						 proxy_connect_timeout 90;
+						 proxy_send_timeout 300;
+						 proxy_read_timeout 90s;
+						 proxy_pass http://faceliftfe:80;
```
Comment (dvaldivia) [inline cluster/nginx.go]: can we call the service `frontend` ?

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: cluster/nginx.go (hunk 2 above)
Comment (dvaldivia) [reply cluster/nginx.go]: `portal`

Comment (dvaldivia) [reply cluster/nginx.go]: `app-router`

File: k8s/deployments/m3-portal-backend-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (dvaldivia) [inline k8s/deployments/m3-portal-backend-deployment.yaml:1]: I think you can do this from `m3 setup`

File: k8s/deployments/m3-portal-proxy.yaml [hunk 4]
```diff
@@ -0,0 +1,107 @@
+---
```
Comment (dvaldivia) [inline k8s/deployments/m3-portal-proxy.yaml:1]: I think you should do this from `m3 setup`

**Review by dvaldivia — CHANGES_REQUESTED** [review]

File: cluster/nginx.go (hunk 1 above)
Comment (Alevsk) [reply cluster/nginx.go:253]: This is the nginx.conf file of `nginx-resolver`, we don't need to handle `/api/` here, request are going to be forwarded to `http://m3-portal-proxy:80`, that proxy load balancer knows to route traffic between frontend and backend.

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**